client/prisma/schema.prisma

```diff
@@ -0,0 +1,200 @@
+  createdAt       DateTime          @default(now())
+  updatedAt       DateTime          @updatedAt
+  deletedAt       DateTime? //optional field
+  links           Links[] //this field is only used for querying relation not implemented in database level
+  companyOwned    Companies[] //this field is only used for querying relation not implemented in database level
+  jobs            Jobs[] //this field is only used for querying relation not implemented in database level
+  gigs            Gigs[] //this field is only used for querying relation not implemented in database level
+  keyWords        UserKeyword[] //this field is only used for querying relation not implemented in database level
+  companies       CompanyMembers[] //this field is only used for querying relation not implemented in database level
+  JobApplications JobApplications[] //this field is only used for querying relation not implemented in database level
+  GigApplications GigApplications[]
+}
+
+model Links {
+  id               String    @id @default(cuid())
+  linkableId       String    @db.Uuid
+  linkableType     String    @db.VarChar
+  title            String    @db.VarChar
+  type             String    @db.VarChar
+  url              String    @unique @db.VarChar
+  featuredPhotoUrl String    @unique @db.VarChar
+  createdBy        String
+  createdAt        DateTime  @default(now())
+  updatedAt        DateTime  @updatedAt
+  deletedAt        DateTime? //optional field
+  creator          Users     @relation(fields: [createdBy], references: [id]) //this field is only used for querying relation not implemented in database level
+}
+
+model Companies {
+  id            String           @id @default(cuid())
+  name          String           @db.VarChar
+  email         String           @db.VarChar
+  description   String
+  logoUrl       String?          @db.VarChar //optional field
+  coverPhotoUrl String?          @db.VarChar //optional field
+  createdBy     String
+  createdAt     DateTime         @default(now())
+  updatedAt     DateTime         @updatedAt
+  deletedAt     DateTime? //optional field
+  creator       Users            @relation(fields: [createdBy], references: [id])
+  jobs          Jobs[]
+  members       CompanyMembers[]
+}
+
+model Jobs {
+  id               String            @id @default(cuid())
+  companyId        String
+  company          Companies         @relation(fields: [companyId], references: [id])
+  title            String            @db.VarChar
+  slug             String            @db.VarChar //required for detailview of the job
+  description      String
+  position         String?           @db.VarChar //optional field
+  opportunity      String?           @db.VarChar //optional field
+  featuredPhotoUrl String?           @db.VarChar //optional field
+  status           String?           @db.VarChar //suggestion: Enum field sholuld be more suitable here because we know the possible outcomes
```

Comment:
Yes let's go with an enum. I just didn't go with an enum from the initial design because I think that we might change it up in the future and it would be a pain to migrate it via just PostgreSQL.

But while we are using Prisma so it shouldn't be the case, I think. But let me know your thoughts about it.

And definitely let's go with an enum. The enum values are in TypeScript models in the frontend application.